src/gobprepare/prepare_client.py

```diff
@@ -43,7 +43,7 @@
 
 READ_BATCH_SIZE = 100000
 WRITE_BATCH_SIZE = 100000
-
+BRP_CATALOGUE = "brp"
```

Comment:
I don't think this is used anymore

Reply:
👍🏻! verwerkt.
linter klaagt niet hierover!